jenkins/bb-fuel-minimal.groovy

```diff
@@ -32,7 +32,7 @@ pipeline {
         string(name: 'IDENTITY_CLIENT', defaultValue: 'hybrid-flow', description: 'Identity client')
         string(name: 'BB_FUEL_VERSION', defaultValue: 'latest', description: '')
         booleanParam(name: 'PRERELEASE', defaultValue: false, description: 'Only applicable if BB_FUEL_VERSION = latest')
-        booleanParam(name: 'HEALTHCHECK_USE_ACTUATOR', defaultValue: true, description: 'Healthcheck endpoint is changed since 2.17.0 release')
+        booleanParam(name: 'HEALTHCHECK_USE_ACTUATOR', defaultValue: true, description: 'Should be true if BOM version => 2.17.0')
```

Comment:
true is default, having a message `"Should be false if DBS version is before 2.17.0"` makes more sense here

Reply:
Done.